frontend/src/views/WatchlistsDetails.vue

```diff
@@ -6,104 +6,231 @@
 
     <DataTable
       :config="config"
-      :rows="sortedRows"
+      :rows="sortedItems"
     >
       <template
-        #field(name)="{ row }"
+        #beforeFilters(right)
+      >
+        <AssetSearch
+          v-model="selectedAsset"
+          placeholder="Add Symbol"
+        />
+      </template>
+
+
+      <!--Copied from Transactions.vue-->
+      <template
+        #field(ticker)="{ value , row }"
       >
         <AppLink
-          :ticker="row.symbol"
+          v-if="row.symbol"
+          :ticker="value"
+          class="font-bold"
           to="ticker_details"
         >
-          <p class="font-normal text-sm">
-            {{ row.name }}
+          <p class="mr-3 font-bold">
+            {{ row.symbol.symbol }}
           </p>
         </AppLink>
+        <span v-else>
+          {{ row.symbol.symbol }}
+        </span>
       </template>
+
       <template
-        #field(symbol)="{ row }"
+        #field(name)="{value, row }"
       >
         <AppLink
-          :ticker="row.symbol"
+          v-if="row.symbol"
+          :ticker="value"
+          class="font-bold"
           to="ticker_details"
         >
           <p class="font-normal text-sm">
-            {{ row.symbol }}
+            {{ row.symbol.name }}
           </p>
         </AppLink>
+        <span v-else>
+          {{ row.symbol.name }}
+        </span>
+      </template>
+
+      <template
+        #field(dayPLChange)="{ row}"
+      >
+        <AssetPriceChange
+          v-if="row.lastPrice"
+          :compare-from="row.lastPrice.previousClose"
+          :compare-to=" row.lastPrice.price"
+        />
+        <span v-else>
+          --
+        </span>
+      </template>
+
+      <template
+        #actions="{row}"
```

Comment:
The actions column is displayed on the next line, i suppose there is something not quite right with the grid template columns

Reply:
Yeah, i noticed, it happened only after #208

Reply:
I can fix this later in the DataTable